org.eclipse.winery.workflowmodeler/src/app/model/interface.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2017 ZTE Corporation.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and the Apache License 2.0 which both accompany this distribution,
+ * and are available at http://www.eclipse.org/legal/epl-v10.html
+ * and http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Contributors:
+ *     ZTE - initial API and implementation and/or initial documentation
+ */
+
+import {Operation} from './operation';
+
+export class Interface {

```

Comment:
Yes, it's a bad practice. In fact the class never be used.  I will delete the class.

I created the class 'Interface' for there is a property called 'interface' in the former data structure. To be align with the former data structure, the property 'interface' will still be used but not by the form of class 'Interface'.